webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5046,6 +5048,10 @@ const firstVisibleMsg = {
                     'km-csat-close-button'
                 ).onclick = function (e) {
                     e.preventDefault();
+                    kmLocalStorage.setItemToLocalStorage(
+                        CHAT_GROUP_ID, { isConversationClosed : true }
```

Comment:
is the 'conversation' getting closed today, or its simply about user closing the CSAT modal?